streamparse/util.py

```diff
@@ -182,7 +182,8 @@ def get_config(config_file=None):
         with open("config.json") as fp:
             config = json.load(fp)
     else:
-        config = json.load(config_file)
+        with open(config_file) as fp:
```

Comment:
As per the docstring, `config_file` in this case is supposed to be a file-like object. Is there an issue you've encountered that you're trying to fix here?

Reply:
you right, i did not read docstring